.github/workflows/release.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
 
       - name: Trigger Manpage Generation
-        uses: ./.github/workflows
```

Comment:
Did you check this in your repo? Do we know if this works?

Reply:
Looking at the docs it seems like we're using it incorrectly still

https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow

Reply:
ohh I didn't check to see if it worked in my repo first. I do know that the last commit that I did was passing on my repo

Reply:
To test this the easiest way would be to do it in your PR with the trigger for `release.yml` changed to 

```
on:
  push
```